release-notes-5.4.0.md

```diff
@@ -5,4 +5,7 @@ ${version-number}
 
 #### New Features
 
+#### Bug fixes
+ - [SCMOD-13334](https://portal.digitalsafe.net/browse/SCMOD-13334) [Tomcat Base Image Issue #41](https://github.com/CAFapi/opensuse-tomcat-image/issues/41): Tomcat-Juli Jar Causes java.lang.NoClassDefFoundError: Could not initialize class org.postgresql.Driver
```

Comment:
This is an open source repo so no internal JIRA links should be in the release notes.

Reply:
I've removed the link